docs/code-and-learn-agenda.md

```diff
@@ -0,0 +1,107 @@
+
+- The triage team: this team focuses on helping answer the many questions which come into the project issue tracker, as well as how to best organize answers and update docs
+- The security working group: this group is in charge of helping us define and maintain our security model, docs, and reporting
+
+Additionally we have roles for folks who are making technical contributions:
+
+- Contributor: these are folks who have made a few contributions to code and are given commit access to land PRs and their own work
+- Repo Captains: these are folks who where contributors on a given repo but have demonstrated strong judgement and reliability who are also given publish permissions and more
+  responsibility within the given repo
+
+And finally we have the leadership group, which is the Technical Committee or as we call it, the TC. This group oversees the larger project direction and is there to help resolve conflicts or make
+important decisions.
+
+## Questions
+
+Does anyone have questions on the structure of the project?
+
+## Project Goals (10 - 15 min)
+
+Because the project is made up of individual volunteers with many different skills, we attempt to organize the priorities and work with a distributed consensus model. This means
+the goals are yours to set. That said, currently because the project is undergoing a revitalization effort, we are maintaining a central list of goals in the `discussions`
+repository.
+
+### General Goals
+
+https://github.com/expressjs/discussions
+
+### Express 5 Goals
+
+https://github.com/expressjs/discussions/issues/266
+
+### Express 6 Goals
+
+https://github.com/expressjs/discussions/issues/267
+
+### Questions
+
+Anyone have questions before we start contributing?
+
+## Lets Start Contributing (1-1.5h)
+
+We picked out three specific goals the project has which we think will make simple yet very helpful contributions today:
+
+1. Documentation updates for v5: https://github.com/expressjs/expressjs.com/
+1. Updating dependencies to use `^`: ranges: https://github.com/expressjs/discussions/pull/290
+1. Adding engines fields to packages https://github.com/expressjs/discussions/pull/289
+1. Remove compat deps: ex. https://github.com/pillarjs/router/blob/master/package.json
+1. Triage issues: https://github.com/expressjs/express/blob/master/Triager-Guide.md
+
+
+### Demo one of each
+
+1. Update a dep, show how to test it locally, then show how to push it up and open a PR.
+- (router -> parseurl) https://github.com/pillarjs/router/blob/878b0e02365047f34c66b4a252e91597f8056635/package.json
+
+2. Update an engines field. Opening the PR and testing already demo'd.
+- (router) https://github.com/pillarjs/router/blob/master/package.json
+
+3. Update a doc. Show how to run locally. Open a PR and show the preview URL.
+  - https://github.com/expressjs/expressjs.com?tab=readme-ov-file#local-setup-using-docker
+
+4. Triage an issue.
+- (bad) https://github.com/jshttp/cookie/pull/202
```

Comment:
What do you mean by bad?

Reply:
In this specific case, it's a PR that was meant for their own fork, and mistakenly was sent to the source repo.

Reply:
Yeah, I should clean up the language here. I was just hurrying to push what I had so that we would have something to share even if incomplete. The consequence of working on it right before the event. :P